applications/logging-operator/6.0.3/defaults/cm.yaml

```diff
@@ -107,29 +109,29 @@ data:
       statefulsetAnnotations:
         secret.reloader.stakater.com/reload: logging-operator-logging-fluentd-tls
     fluentbit:
-      podPriorityClassName: "dkp-critical-priority"
       image:
         # Explicitly specify the version here. This should be updated when logging-operator is upgraded.
         # Also, update the image in fluent-bit configuration if the image is upgraded here.
         repository: ghcr.io/mesosphere/dkp-container-images/docker.io/fluent/fluent-bit
-        tag: 3.2.5-d2iq.0
+        tag: 4.0.8
```

Comment:
This image version doesn't exist in `dkp-container-images` repository - https://github.com/mesosphere/dkp-container-images/pkgs/container/dkp-container-images%2Fdocker.io%2Ffluent%2Ffluent-bit